lib/Frontend/Frontend.cpp

```diff
@@ -74,17 +74,51 @@ void CompilerInstance::setPrimarySourceFile(SourceFile *SF) {
 bool CompilerInstance::setup(const CompilerInvocation &Invok) {
   Invocation = Invok;
 
+  setUpLLVMArguments();
```

Comment:
Next time, please do this refactoring separately from the InputFile stuff. There's basically two separate changes to review here.

Reply:
Sorry!